src/current/_includes/v23.1/sidebar-data/cloud-deployments.json

```diff
@@ -131,28 +131,28 @@
             ]
             },
             {
-            "title": "CMEK for Dedicated",
+            "title": "Customer-Managed Encryption Keys (CMEK)",
             "items": [
                 {
-                "title": "Manage Customer-Managed Encryption Keys (CMEK) for Dedicated",
+                "title": "Manage CMEK for Dedicated",
                 "urls": [
                     "/cockroachcloud/managing-cmek.html"
                 ]
                 },
                 {
-                "title": "Provision AWS KMS Keys and IAM Roles for CMEK",
+                "title": "Provision AWS for CMEK",
                 "urls": [
                     "/cockroachcloud/cmek-ops-aws.html"
                 ]
                 },
                 {
-                "title": "Provision GCP KMS Keys and Service Accounts for CMEK",
+                "title": "Provision GCP for CMEK",
                 "urls": [
                     "/cockroachcloud/cmek-ops-gcp.html"
                 ]
                 },
                 {
-                "title": "Customer-Managed Encryption Keys (CMEK)",
+                "title": "Customer-Managed Encryption Keys (CMEK) Overview",
                 "urls": [
```

Comment:
On the overview page, looks like there are a few links to pages where the link references is not updated to the new title. This isn't a big deal and doesn't necessarily have to be tackled in this PR, but thought I would highlight!